2013-05-23	Nicolas Setton  <setton@adacore.com>

	* gtk/gtkclipboard-quartz.c: Replace undefined symbol.

--- ../gtk+-3.8.2-orig/gtk/gtkclipboard-quartz.c	2013-05-13 12:15:07.000000000 +0200
+++ gtk/gtkclipboard-quartz.c	2013-05-23 14:40:04.819631640 +0200
@@ -108,7 +108,7 @@
   memset (&selection_data, 0, sizeof (GtkSelectionData));
 
   selection_data.selection = clipboard->selection;
-  selection_data.target = _gtk_quartz_pasteboard_type_to_atom (type);
+  selection_data.target = gdk_quartz_pasteboard_type_to_atom_libgtk_only (type);
   selection_data.display = gdk_display_get_default ();
   selection_data.length = -1;
 
--- gdk/quartz/gdkwindow-quartz.c.old	2012-08-06 15:33:18.000000000 +0200
+++ gdk/quartz/gdkwindow-quartz.c	2012-08-06 15:40:51.000000000 +0200
@@ -861,10 +861,11 @@
     {
       NSRect rect = [[screens objectAtIndex:i] frame];
 
+      screen = [screens objectAtIndex:i];
+
       if (x >= rect.origin.x && x <= rect.origin.x + rect.size.width &&
           y >= rect.origin.y && y <= rect.origin.y + rect.size.height)
         {
-          screen = [screens objectAtIndex:i];
           break;
         }
     }
